contrib/chinese/config.el

```diff
@@ -0,0 +1,32 @@
+;;; config.el --- Colors Layer configuration File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;; Variables
+
+(defvar chinese-im-enable-pinyin nil
+  "Use Pinyin input method.")
+
+(defvar chinese-im-enable-wubi nil
+  "Use Wubi input method.")
```

Comment:
No, only one input method could be enabled at a time. I will take your advice to unify the variable to `chinese-default-input-method`. Thanks.